README.md

```diff
@@ -39,21 +39,21 @@ The Docker installation package consists of two independent products which insta
   
 It is easy to confuse which system you are currently working with.
   
-Docker Desktop has a builtin daemon which works in parallel with the `dockerd` daemon.
+Docker Desktop has a built-in daemon which works in parallel with the `dockerd` daemon.
   
-Docker Desktop keep it's settings and storage of images and containers in the user's home directory.
+Docker Desktop keeps its settings and storage of images and containers in the user's home directory.
   
-Docker Engine keep it's settings and the storage in the system directories.
+Docker Engine keeps its settings and the storage in the system directories.
   
 Docker Desktop works with userns-remap turned on only. It is hardcoded and can't be changed.
   
-Docker Desktop has builtin daemon which works in parallel with the dockerd daemon.   
+Docker Desktop has a built-in daemon which works in parallel with the `dockerd` daemon.   
 
 Therefore, to successfully work with DUNE you should:
 
->+ Clone latest DUNE repository.
+>+ Clone the latest DUNE repository.
 >+ Rebuild the image by calling ./bootstrap.sh.
->+ Add a root directory of host system to list of shared directories in Docker Desktop settings.
+>+ Add a root directory of the host system to the list of shared directories in Docker Desktop settings.
```

Comment:
Shouldn't this be "Add a directory..."?  In a Unix-style system there's only one root directory and it's not customarily shared with containers.

Reply:
It really does require the entire filesystem to be shared with the container?

In that case, the wording should be "Add the root directory..."

And yes that's awful.

Reply:
> It really does require the entire filesystem to be shared with the container?

I guess this is not known exactly until the end of development. So at the beginning they decided to share all to don't limit themselves in case they will need to have access somewhere in the file system.

> In that case, the wording should be "Add the root directory..."
> And yes that's awful.

IMHO it will be possible to change when we will release it and get feedback from real users. We will know better what is really need and what is not. Then it will be easier to make decision about limiting accesses, rights and so on.

So now, if you have no other remarks, I will be glad if you approve this PR and I will merge it.

Reply:
> I guess this is not known exactly until the end of development. So at the beginning they decided to share all to  don't limit themselves in case they will need to have access somewhere in the file system.

In that case, the text needs to change as I suggested.  Sharing the entirety of the filesystem is so unusual that the ambiguity needs to be removed.  I assumed something else was meant because it's so unusual.  I even suggest adding a disclaimer sentence, incorporating your explanation in this thread into the document.  I assure you I won't be the only person questioning the document if there's no explanation.

Reply:
> In that case, the text needs to change as I suggested. Sharing the entirety of the filesystem is so unusual that the ambiguity needs to be removed. I assumed something else was meant because it's so unusual. I even suggest adding a disclaimer sentence, incorporating your explanation in this thread into the document. I assure you I won't be the only person questioning the document if there's no explanation.

Let's solve problems as they come. I decided to add this PR because I was confused by architecture of Docker. I was solving of problem with files shared between the host and the container and I was sure that Docker Desktop is just a frontend over `dockerd` daemon. I was very surprised to found that Docker Desktop works even if the `dockerd` is shot down. So I decided to highlight this information for the users of DUNE to save their time.

If someone asks about why do they need to share whole root with the container I will write additional topic which explain this subject in separated PR. IMHO it is better so solve one issue in one PR and merge them separately.